sky/resources.py

```diff
@@ -56,7 +56,7 @@ def __init__(
         accelerator_args: Optional[Dict[str, str]] = None,
         use_spot: Optional[bool] = None,
         spot_recovery: Optional[str] = None,
-        region: Optional[str] = None,
+        region: Optional[Union[str, List[str]]] = None,
```

Comment:
As mentioned above, we may not want to add an additional API for `Resources`. Why not do similar thing as `accelerators`, i.e., when user specifies multiple regions, we create multiple `Resources` object that has a single region specified?

Reply:
The reason is because when the user specifies a list of accelerators, and a list of regions, we are not sure if we want to do a cartesian product between the two

Reply:
removed regions from this PR